pkg/instancetype/apply/gpu_test.go

```diff
@@ -0,0 +1,56 @@
+//nolint:dupl
+package apply_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	virtv1 "kubevirt.io/api/core/v1"
+	v1beta1 "kubevirt.io/api/instancetype/v1beta1"
+
+	"kubevirt.io/kubevirt/pkg/instancetype/apply"
+	"kubevirt.io/kubevirt/pkg/libvmi"
+)
+
+var _ = Describe("instancetype.Spec.GPUs", func() {
+	var (
+		vmi            *virtv1.VirtualMachineInstance
+		preferenceSpec *v1beta1.VirtualMachinePreferenceSpec
+
+		vmiApplier       = apply.NewVMIApplier()
+		field            = k8sfield.NewPath("spec", "template", "spec")
+		instancetypeSpec = &v1beta1.VirtualMachineInstancetypeSpec{
+			GPUs: []virtv1.GPU{
+				{
+					Name:       "barfoo",
+					DeviceName: "vendor.com/gpu_name",
+				},
+			},
+		}
```

Comment:
nit - usually variables are initialized in `BeforeEach()` so we can be sure that no state is transferred between `It` blocks. But in this case, these variables are not changed in thests.
You can keep it as is.

Reply:
Yes, this variable is not modified during the test